frontend/packages/schema-editor/src/components/SchemaInspector/restrictions/NumberRestrictionsReducer.test.tsx

```diff
@@ -310,15 +310,15 @@ describe('NumberRestrictionsReducer', () => {
         };
         it('Returns InvalidMaxMinValue if min > max', () => {
           const result = validateMinMax({ ...bothExclState, min: 7, max: 5 });
-          expect(result).toBe(NumberRestrictionsError.InvalidMaxMinValue);
+          expect(result).toBe(NumberRestrictionsError.IntervalMustBeLargeEnough);
```

Comment:
`IntervalMustBeLargeEnough` er forsåvidt en riktig feilmelding her, men i denne situasjonen gir det mer mening for brukeren å få beskjed om at min må være mindre enn maks.
```suggestion
          expect(result).toBe(NumberRestrictionsError.MinMustBeLessThanMax);
```